includes/Services/FlowService.php

```diff
@@ -0,0 +1,191 @@
+<?php
+namespace NewfoldLabs\WP\Module\Onboarding\Services;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+use NewfoldLabs\WP\Module\Onboarding\Data\Flows;
+use NewfoldLabs\WP\Module\Onboarding\Data\Data;
+
+class FlowService {
+
+	// An array to keep the Flow Data updated in the database
+	protected $updated_flow_data = array();
+
+	public static function initalize_flow_data() { 
+		$default_flow_data = Flows::get_data();
+        if ( ! ( $flow_data = self::read_flow_data_from_wp_option() ) )
+            return self::get_default_flow_data();
```

Comment:
```suggestion
        if ( ! ( $flow_data = self::read_flow_data_from_wp_option() ) )
            return self::get_default_flow_data();
        $default_flow_data = Flows::get_data();
```
Let's not read the default flow data until it is necessary.